server/app/routes/lecture.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
+var nodemailer = require('nodemailer');
 var router = require('express').Router();
 var db = require('../../database');
 var Lecture = db.model('lecture');
+var Question = db.model('question');
+
+var transporter = nodemailer.createTransport({
```

Comment:
If email is or might become an action more widely used in your backend, consider making an "email utility" module internally.